Models/Board.cs

```diff
@@ -1,47 +1,54 @@
 ﻿using Chess_Backend.Models.Pieces;
 using Chess_Backend.Models.Positions;
-using System;
-using System.Collections.Generic;
-using System.Text;
-
 namespace Chess_Backend.Models
 {
     public class Board : IBoard
     {
         public List<Piece> Pieces { get; private set; }
-        public Dictionary<Tile, Piece> Position { get; private set; }
+        public Dictionary<Tile, Piece> Positions { get; private set; }
+        public Color CurrentTurnColor { get; private set; }
 
-        public Board(List<Piece> pieces)
+        public Board()
```

Comment:
why do you need the empy ctor? you don't use loc to generate the object for you anymore